src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/BuildIncrementalismTest.cs

```diff
@@ -89,7 +89,7 @@ public async Task RazorGenerate_RegeneratesTagHelperInputs_IfFileChanges()
 
             // Assert - 2
             Assert.BuildPassed(result);
-            Assert.FileContentEquals(result, tagHelperOutputCache, "[]");
```

Comment:
This broke when I added the missing tag helper descriptor providers. The file isn't empty since this project references the component shim, and that lights up some built-ins.